src/examples/slickgrid/example30.html

```diff
@@ -0,0 +1,67 @@
+<template>
+  <h2>${title}</h2>
+  <div class="subtitle" innerhtml.bind="subTitle"></div>
+
+  <div class="mb-2">
+    <div class="btn-group btn-group-sm" role="group" aria-label="Basic example">
+      <button type="button" class="btn btn-outline-secondary" data-test="toggle-readonly-btn"
+              click.delegate="toggleGridEditReadonly()">
+        <i class="fa fa-table"></i> Toggle Edit/Readonly Grid
```

Comment:
Since I see it here, are you using an older Version of fontawesome? The newer one requires `fas fa-*`

Reply:
Yes I've used the old version `4.7.0`, I never tried the new one, I find the new font too fat and the thin one is on the paid side 😆

Reply:
if you want to contribute and change it to latest Font-Awesome (or Fort Awesome like I think I saw you using), then I wouldn't mind another PR 😉

Reply:
Tbh me neither, but since this not the examples repo I think it doesn't matter. Next time around at the samples I'll bump and change to fortawesome.